Makefile

```diff
@@ -29,6 +29,18 @@ RELEASES ?= v0.8.5 v0.9.0
 SHORT_INTEGRATION_TEST ?=
 CUSTOM_CNI_URL ?=
 
+CGO_ENABLED ?= 0
+GO_EXTRA_BUILDFLAGS ?=
+GO_EXTRA_LDFLAGS ?=
+
+RACE_DETECTOR_ENABLED ?=
```

Comment:
I would call it `WITH_RACE_DETECTOR` or just `WITH_RACE` – because what it essentially does is adds a `-race` flag. In the future, we could add something like `WITH_DEBUG`, `WITH_ASSERTS`, etc.

I also think we should differentiate between internal variables and variables that are expected to be set by a contributor (like `PUSH`, `USERNAME`, etc.). Probably by adding comments.

Reply:
yeah, `WITH_RACE` is shorter. as for the variables -- they are not truly internal, you can set them. internal variables are not overrideable, these vars you can override if you want to test some new build flag.

Reply:
I mean, those variables _can_ be overridden, but only a subset of them are _expected_ to be. For example, `PUSH` and `USERNAME` are expected to be overridden by contributor – to be able to contribute. Something like `GO_EXTRA_BUILDFLAGS` – not really. So `PUSH` and `USERNAME` should have comments and/or (preferably "and" :) ) be mentioned in `make help` output, but `GO_EXTRA_BUILDFLAGS` shouldn't.

Reply:
yep, agree, I mentioned `WITH_RACE` there